docs/UserGuide.md

```diff
@@ -16,27 +16,29 @@ Broadly speaking, ContaX consists of an *Address Book* for managing Contacts, an
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `ContaX.jar` from [here](https://github.com/AY2122S2-CS2103-W17-1/tp/releases).
+2. Download the latest `ContaX.jar` from [here](https://github.com/AY2122S2-CS2103-W17-1/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for ContaX.
+3. Copy the file to the folder you want to use as the _home folder_ for ContaX.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
```

Comment:
I would suggest to change it to running via CLI, because if use double-click on Mac, the directory folders get all messed up.

Reply:
Fixed in commit 89fc2cd.

The latest one looks like this:
![image](https://user-images.githubusercontent.com/13163634/160393451-39861ffa-947e-4b6e-b993-301907bfad16.png)